### ISSUE TITLE
Can't use on gitpod

### ISSUE THREAD

**Issue by bitsnaps** [silvanmelchior/IncognitoPilot#43]
Hi,

Trying different combination (from Docker, from source...), I keep getting the error:
```
Error: Could not authenticate to backend. This probably means there is no or an invalid authentication token provided in the URL. Please check the startup console output of the backend and add a valid token to the URL.
```
You can find the Gitpod config [here](https://github.com/bitsnaps/IncognitoPilot).

**Comment by silvanmelchior** [comment]
Hi, does the URL you enter into your browser contain the token (i.e. localhost://3030#token=something)? The terminal shows you the link you have to use

**Comment by bitsnaps** [comment]
> Hi, does the URL you enter into your browser contain the token (i.e. localhost://3030#token=something)? The terminal shows you the link you have to use

Yep, it does, I've also tried to enter it manually in the URL, i've provided you the reproducible example in the link above, and to be honest with you; this can be a security measure by Gitpod, which prevent users to share workspaces for malicious actions, I wish the app shows more useful debugging informations so users can dig deeper.

**Comment by silvanmelchior** [comment]
Hi, sorry for the late reply, I was on vacation.

Unfortunately, I haven't used gitpod yet, so I can't really support. The message you describe is usually shown if the auth token was either not found or if so, couldn't be verified by the backend.

One way of debugging could be to have it running locally & on gitpod, and then use the browser developer tools to compare the messages sent between frontend and backend.
